src/base-importer/BaseImporter.js

```diff
@@ -0,0 +1,358 @@
+import RequestContext, { FileReference, Auth } from 'api-flow'
+
+import {
+    registerImporter,
+    DynamicValue,
+    DynamicString
+} from '../paw-mocks/PawShims'
+
+@registerImporter // eslint-disable-line
+export default class BaseImporter {
+    static identifier = 'com.luckymarmot.PawExtensions.BaseImporter';
+    static title = 'Base Importer';
```

Comment:
Same for these static properties. They shouldn't be here and we should let the concrete classes set them.